### ISSUE TITLE
Create a Dictu Playground

### ISSUE THREAD

**Issue by Jason2605** [dictu-lang/Dictu#262]
# Dictu Playground
Something that would be a nice addition would be an online playground where people can try Dictu in their browser without the need to go through the installation process, an online REPL of sorts. The main consideration here would be safety as it would be executing potentially unsafe code.

**Comment by avinashupadhya99** [comment]
Hey, I am interested in doing this. Do you have any communication channels like Discord or Slack where we can further communicate?
If not, which language are thinking about implementing this in?

**Comment by Jason2605** [comment]
Yep you could probably find me on discord, what is your name / discrim? Happy to discuss further there, and then once we have a solid action plan it would be good for visibilities sake to update this ticket if/when we need to.

**Comment by avinashupadhya99** [comment]
My discord username is avinashupadhyaya

**Comment by Jason2605** [comment]
If we can get a bare WASM version running for a playground I'd be more than happy with that
